src/Microsoft.ML.FastTree/TreeEnsemble/InternalRegressionTree.cs

```diff
@@ -409,7 +409,7 @@ protected void Save(ModelSaveContext ctx, TreeType code)
             writer.WriteDoubleArray(_previousLeafValue);
         }
 
-        public virtual void Save(ModelSaveContext ctx)
+        internal virtual void Save(ModelSaveContext ctx)
```

Comment:
>internal [](start = 8, length = 8)

Since this is already an internal class there is no point in having this be internal I think.

Reply:
Sounds good!

---
In reply to: [254471904](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254471904) [](ancestors = 254471904)